tests/test_session.py

```diff
@@ -280,6 +290,15 @@ def test_load2():
         _ = Session(file_path="session_test10.json")
 
 
+def test_load3():
+    session = Session()
+    with pytest.raises(FileNotFoundError, match=r"Invalid path: must be a string and refer to an existing location. Given path: session_test10.json"):
```

Comment:
This test case is not suitable because there is no file to load.

Reply:
fixed, I added a new try/except to the Session so we can have the same test cases for there as well.